packages/allure-playwright/src/helpers.ts

```diff
@@ -9,6 +9,16 @@ export interface Metadata {
 }
 
 export class allure {
+ /** 
+  * add environment info for test
+  * @default process.env
+  */
+  static environment(environment: Record<string, string | undefined> = process.env) {
+    test.info().attach("environment.json", {
```

Comment:
You don't need separate file for enviorment info. Instead you can just use `addMetadataAttachment` and add new metadata filed to it. It will be much easier to parse.

Reply:
not agree with you, since allure generates `environment.json` as a separate file, I assume that is a good way to understand which environments are coming.